### ISSUE TITLE
cargo instal error: no method named `clone` found for type `syn::UseTree` in the current scope

### ISSUE THREAD

**Issue by xlc** [hobofan/cargo-nono#44]
`cargo install cargo-nono` gives me this error:

```
error[E0599]: no method named `clone` found for type `syn::UseTree` in the current scope
   --> /Users/xiliangchen/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-nono-0.1.5/src/check_source.rs:211:55
    |
211 |                         item_tree: use_statement.tree.clone(),
    |                                                       ^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: failed to compile `cargo-nono v0.1.5`, intermediate artifacts can be found at `/var/folders/l6/p9dxwd1x5j11q2xnyg7m_xpc0000gn/T/cargo-installica1OU`

Caused by:
  Could not compile `cargo-nono`.

To learn more, run the command again with --verbose.
```

```
$ cargo -V
cargo 1.37.0 (9edd08916 2019-08-02)
$ rustc -V
rustc 1.37.0 (eae3437df 2019-08-13)
```

**Comment by hobofan** [comment]
Could you try it with the current master:
```
cargo install --git https://github.com/hobofan/cargo-nono.git
```

**Comment by xlc** [comment]
works now. thanks

**Comment by hobofan** [comment]
Thanks for reporting! I just published version `0.1.6` so the default cargo install should also work again.
